corehq/apps/app_manager/tests/data/suite/schedule-entry.xml

```diff
@@ -6,11 +6,20 @@
+            <anchor_a983e9 function="edd"/>
+            <last_visit_number_a983e9 function="last_visit_number_a983e9"/>
+            <last_visit_date_a983e9 function="last_visit_date_a983e9"/>
+            <next_a983e9 function="if(current_schedule_phase = '' and $anchor_a983e9 != '' and (today() &gt;= (date($anchor_a983e9) + int(instance('schedule:m1:p1:f1')/schedule/@starts)) and (instance('schedule:m1:p1:f1')/schedule/@expires = '' or today() &lt;= (date($anchor_a983e9) + int(instance('schedule:m1:p1:f1')/schedule/@expires)))), date($anchor_a983e9) + int(coalesce(instance('schedule:m1:p1:f1')/schedule/visit[if(@repeats = 'True', @expires = '' or today() &lt;= (date($last_visit_date_a983e9) + int(@increment) + int(@expires)), @expires = '' or today() &lt;= (date($anchor_a983e9) + int(@due) + int(@expires)))][1]/@due, 2147483647)), if(current_schedule_phase = 1 and $anchor_a983e9 != '' and (today() &gt;= (date($anchor_a983e9) + int(instance('schedule:m1:p1:f1')/schedule/@starts)) and (instance('schedule:m1:p1:f1')/schedule/@expires = '' or today() &lt;= (date($anchor_a983e9) + int(instance('schedule:m1:p1:f1')/schedule/@expires)))), date($anchor_a983e9) + int(coalesce(instance('schedule:m1:p1:f1')/schedule/visit[$last_visit_number_a983e9 = '' or @id &gt; $last_visit_number_a983e9][if(@repeats = 'True', @expires = '' or today() &lt;= (date($last_visit_date_a983e9) + int(@increment) + int(@expires)), @expires = '' or today() &lt;= (date($anchor_a983e9) + int(@due) + int(@expires)))][1]/@due, 2147483647)), 2147483647))"/>
+            <anchor_39f0ea function="dob"/>
+            <last_visit_number_39f0ea function="last_visit_number_39f0ea"/>
+            <last_visit_date_39f0ea function="last_visit_date_39f0ea"/>
+            <next_39f0ea function="if(current_schedule_phase = 2 and $anchor_39f0ea != '' and (today() &gt;= (date($anchor_39f0ea) + int(instance('schedule:m1:p2:f0')/schedule/@starts)) and (instance('schedule:m1:p2:f0')/schedule/@expires = '' or today() &lt;= (date($anchor_39f0ea) + int(instance('schedule:m1:p2:f0')/schedule/@expires)))), date($anchor_39f0ea) + int(coalesce(instance('schedule:m1:p2:f0')/schedule/visit[$last_visit_number_39f0ea = '' or @id &gt; $last_visit_number_39f0ea][if(@repeats = 'True', @expires = '' or today() &lt;= (date($last_visit_date_39f0ea) + int(@increment) + int(@expires)), @expires = '' or today() &lt;= (date($anchor_39f0ea) + int(@due) + int(@expires)))][1]/@due, 2147483647)), 2147483647)"/>
+            <next_due function="date(min($next_हिन्दी,$next_a983e9,$next_39f0ea))"/>
+            <is_late function="$next_due &lt; today()"/>
```

Comment:
:imp: 
This was the bane of my existence for a while. I should have extracted that statement from here and composed it directly in the test. But... it's done now.